lib/spack/spack/compilers/nag.py

```diff
@@ -20,6 +21,16 @@ class Nag(Compiler):
                    'f77' : 'nag/nagfor',
                    'fc'  : 'nag/nagfor' }
 
+    @property
+    def openmp_flag(self):
+        return "-openmp"
+
+    @property
+    def cxx11_flag(self):
+        tty.die("cxx11_flag() is not implemented for nag. Consider creating a pull-request.")
+        return "-std=c++11"
```

Comment:
you mean the message? I just don't know what the flags for those compilers, but for consistency i think these properties should be defined in each compiler.